bin/config.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+REPO_DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd)"
+FONZIE_UNSET_USER=0
+
+# _set_user: set (or unset) default user/group id used to run docker commands
+#
+# usage: _set_user
+#
+# You can override default user (and/or group) ID (the current host user/group
+# ID), by defining the FONZIE_USER_ID (and/or FONZIE_GROUP_ID) environment
+# variable.
+#
+# To avoid running docker commands with a custom user/group ID, please set the
+# $FONZIE_UNSET_USER environment variable to 1.
+function _set_user() {
+
+    if [ $FONZIE_UNSET_USER -eq 1 ]
+    then
+        USER_ID=""
+        GROUP_ID=""
+        return
+    fi
+
+    USER_ID=$FONZIE_USER_ID
+    if [ -z $USER_ID ]
+    then
+        USER_ID=$(id -u)
+    fi
```

Comment:
Shorter version of this if :
`USER_ID=${FONZIE_USER_ID:-$(id -u)}`

Reply:
Indeed! Thx.

Reply:
but less readable for bash idiots like me :~(

Reply:
@sampaccoud 
With a comment it's 2 line instead of 4
You can comment : 
# USER_ID=FONZIE_USER_ID or id of the current user if environment var is not set